[2021-06-07 01:32:10,521] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: stage_rox_bike_tables.create_product_table 2021-06-07T01:31:57.496009+00:00 [queued]>
[2021-06-07 01:32:10,529] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: stage_rox_bike_tables.create_product_table 2021-06-07T01:31:57.496009+00:00 [queued]>
[2021-06-07 01:32:10,529] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-07 01:32:10,529] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-07 01:32:10,531] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-07 01:32:10,541] {taskinstance.py:1087} INFO - Executing <Task(PostgresOperator): create_product_table> on 2021-06-07T01:31:57.496009+00:00
[2021-06-07 01:32:10,547] {standard_task_runner.py:52} INFO - Started process 156 to run task
[2021-06-07 01:32:10,549] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'stage_rox_bike_tables', 'create_product_table', '2021-06-07T01:31:57.496009+00:00', '--job-id', '67', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/stage_data_s3_to_redshift.py', '--cfg-path', '/tmp/tmp4nsr4g3b', '--error-file', '/tmp/tmpuwjb5ck6']
[2021-06-07 01:32:10,549] {standard_task_runner.py:77} INFO - Job 67: Subtask create_product_table
[2021-06-07 01:32:10,577] {logging_mixin.py:104} INFO - Running <TaskInstance: stage_rox_bike_tables.create_product_table 2021-06-07T01:31:57.496009+00:00 [running]> on host 21bd802337aa
[2021-06-07 01:32:10,611] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=stage_rox_bike_tables
AIRFLOW_CTX_TASK_ID=create_product_table
AIRFLOW_CTX_EXECUTION_DATE=2021-06-07T01:31:57.496009+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-07T01:31:57.496009+00:00
[2021-06-07 01:32:10,611] {postgres.py:70} INFO - Executing: 
    CREATE TABLE IF NOT EXISTS Product
    (
        ProductID INTEGER NOT NULL,
        Name VARCHAR(50),
        ProductNumber VARCHAR(10),
        MakeFlag INTEGER,
        FinishedGoodsFlag INTEGER,
        Color VARCHAR(20),
        SafetyStockLevel INTEGER,
        ReorderPoint INTEGER,
        StandardCost FLOAT,
        ListPrice FLOAT,
        Size VARCHAR(10),
        SizeUnitMeasureCode VARCHAR(10),
        WeightUnitMeasureCode VARCHAR(10),
        Weight FLOAT,
        DaysToManufacture INTEGER,
        ProductLine VARCHAR(10),
        Class VARCHAR(10),
        Style VARCHAR(10),
        ProductSubcategoryID INTEGER,
        ProductModelID INTEGER,
        SellStartDate TIMESTAMP,
        SellEndDate TIMESTAMP,
        DiscontinuedDate TIMESTAMP,
        rowguid VARCHAR(36),
        ModifiedDate TIMESTAMP,
        PRIMARY KEY (ProductID)
    )
[2021-06-07 01:32:10,617] {connection.py:360} ERROR - Failed parsing the json for conn_id redshift
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/connection.py", line 357, in extra_dejson
    obj = json.loads(self.extra)
  File "/usr/local/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2021-06-07 01:32:10,618] {base.py:78} INFO - Using connection to: id: redshift. Host: doxcluster.craeubgpe2r5.us-east-1.redshift.amazonaws.com, Port: 5439, Schema: dox, Login: doxuser, Password: ***, extra: {}
[2021-06-07 01:32:10,618] {connection.py:360} ERROR - Failed parsing the json for conn_id redshift
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/connection.py", line 357, in extra_dejson
    obj = json.loads(self.extra)
  File "/usr/local/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2021-06-07 01:32:10,618] {connection.py:360} ERROR - Failed parsing the json for conn_id redshift
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/connection.py", line 357, in extra_dejson
    obj = json.loads(self.extra)
  File "/usr/local/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2021-06-07 01:32:10,619] {connection.py:360} ERROR - Failed parsing the json for conn_id redshift
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/connection.py", line 357, in extra_dejson
    obj = json.loads(self.extra)
  File "/usr/local/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2021-06-07 01:32:14,577] {dbapi.py:204} INFO - Running statement: 
    CREATE TABLE IF NOT EXISTS Product
    (
        ProductID INTEGER NOT NULL,
        Name VARCHAR(50),
        ProductNumber VARCHAR(10),
        MakeFlag INTEGER,
        FinishedGoodsFlag INTEGER,
        Color VARCHAR(20),
        SafetyStockLevel INTEGER,
        ReorderPoint INTEGER,
        StandardCost FLOAT,
        ListPrice FLOAT,
        Size VARCHAR(10),
        SizeUnitMeasureCode VARCHAR(10),
        WeightUnitMeasureCode VARCHAR(10),
        Weight FLOAT,
        DaysToManufacture INTEGER,
        ProductLine VARCHAR(10),
        Class VARCHAR(10),
        Style VARCHAR(10),
        ProductSubcategoryID INTEGER,
        ProductModelID INTEGER,
        SellStartDate TIMESTAMP,
        SellEndDate TIMESTAMP,
        DiscontinuedDate TIMESTAMP,
        rowguid VARCHAR(36),
        ModifiedDate TIMESTAMP,
        PRIMARY KEY (ProductID)
    ), parameters: None
[2021-06-07 01:32:15,319] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=stage_rox_bike_tables, task_id=create_product_table, execution_date=20210607T013157, start_date=20210607T013210, end_date=20210607T013215
[2021-06-07 01:32:15,346] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-06-07 01:32:15,383] {local_task_job.py:151} INFO - Task exited with return code 0
